src/Compilers/CSharp/Portable/Symbols/NativeIntegerTypeSymbol.cs

```diff
@@ -82,8 +94,92 @@ internal NativeIntegerTypeSymbol(NamedTypeSymbol underlying) : base(underlying,
 
         internal override NamedTypeSymbol AsNativeInt(bool asNativeInt) => asNativeInt ? this : _underlyingType;
 
-        internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool> isValueTypeOverrideOpt = null) => _underlyingType.Equals(t2, comparison, isValueTypeOverrideOpt);
+        internal override bool Equals(TypeSymbol t2, TypeCompareKind comparison, IReadOnlyDictionary<TypeParameterSymbol, bool>? isValueTypeOverrideOpt = null) => _underlyingType.Equals(t2, comparison, isValueTypeOverrideOpt);
 
         public override int GetHashCode() => _underlyingType.GetHashCode();
+
+        private static ImmutableArray<Symbol> CalculateMembers(NativeIntegerTypeSymbol containingType, NamedTypeSymbol underlyingType)
+        {
+            var underlyingMembers = underlyingType.GetMembers();
+            var builder = ArrayBuilder<Symbol>.GetInstance();
+            addMethodIfAny(SpecialMember.System_Object__GetHashCode);
+            addMethodIfAny(SpecialMember.System_Object__Equals);
+            addMethodIfAny(SpecialMember.System_Object__ToString);
+            return builder.ToImmutableAndFree();
+
+            void addMethodIfAny(SpecialMember id)
+            {
+                var method = (MethodSymbol?)CSharpCompilation.GetRuntimeMember(
+                    underlyingMembers,
+                    SpecialMembers.GetDescriptor(id),
+                    CSharpCompilation.SpecialMembersSignatureComparer.Instance,
+                    accessWithinOpt: null);
+                if (method is object)
+                {
+                    builder.Add(new NativeIntegerMethodSymbol(containingType, method));
+                }
+            }
+        }
+
+        private sealed class NativeIntegerMethodSymbol : WrappedMethodSymbol
```

Comment:
>NativeIntegerMethodSymbol [](start = 29, length = 25)

It looks like container is equal to underlying, but members are not. Should they be?